tasks.py

```diff
@@ -46,40 +49,71 @@ def is_truthy(arg):
 PROJECT_VERSION = PYPROJECT_CONFIG["tool"]["poetry"]["version"]
 
 
-def run_cmd(context, exec_cmd, local=INVOKE_LOCAL):
+def _get_image_name(with_napalm=False):
+    """Gets the name of the container image to use.
+
+    Args:
+        with_napalm (bool): Get name of container image with Napalm installed.
+
+    Returns:
+        str: Name of container image. Includes tag.
+    """
+    if with_napalm and not os.getenv("GITHUB_ACTION", None):
+        name = f"{IMAGE_NAME}:{IMAGE_VER}-{NAPALM_PACKAGE}{NAPALM_VER}"
```

Comment:
Will this scale if we have one for ntc-templates, pyats, etc?

Reply:
looking into poetry's new `groupings` might allow us to do this easier since ultimately this is more of development related issues.

Reply:
Yea, good point. I think we can have a "full" version, and another no-dependency version. This is just for development environment afterall.